src/nomad_measurements/general.py

```diff
@@ -48,7 +48,11 @@
 
 configuration = config.get_plugin_entry_point('nomad_measurements:schema')
 
-m_package = SchemaPackage()
+m_package = SchemaPackage(
+    aliases=[
```

Comment:
What's the purpose of this? Can I now import something from `general.py` like
```py
from nomad_measurements import NOMADMeasurementsCategory
```
?